GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -296,38 +298,41 @@ public static HotkeySettings[] CreateDefaultSettings()
-                    Hk(RevisionGridControl.Commands.ToggleAuthorDateCommitDate, Keys.None),
-                    Hk(RevisionGridControl.Commands.ToggleOrderRevisionsByDate, Keys.None),
-                    Hk(RevisionGridControl.Commands.ToggleShowRelativeDate, Keys.None),
-                    Hk(RevisionGridControl.Commands.ToggleDrawNonRelativesGray, Keys.None),
-                    Hk(RevisionGridControl.Commands.ToggleShowGitNotes, Keys.None),
-                    Hk(RevisionGridControl.Commands.ToggleShowMergeCommits, Keys.Control | Keys.Shift | Keys.M),
-                    Hk(RevisionGridControl.Commands.ToggleShowTags, Keys.Control | Keys.Alt | Keys.T),
-                    Hk(RevisionGridControl.Commands.ToggleHighlightSelectedBranch, Keys.Control | Keys.Shift | Keys.B)),
+                    Hk(RevisionGridControl.Command.CompareSelectedCommits, Keys.None),
+                    Hk(RevisionGridControl.Command.CompareToBase, Keys.Control | Keys.R),
+                    Hk(RevisionGridControl.Command.CompareToBranch, Keys.None),
+                    Hk(RevisionGridControl.Command.CompareToCurrentBranch, Keys.None),
+                    Hk(RevisionGridControl.Command.CompareToWorkingDirectory, Keys.Control | Keys.D),
+                    Hk(RevisionGridControl.Command.CreateFixupCommit, Keys.Control | Keys.X),
+                    Hk(RevisionGridControl.Command.GoToChild, Keys.Control | Keys.N),
+                    Hk(RevisionGridControl.Command.GoToCommit, Keys.Control | Keys.Shift | Keys.G),
+                    Hk(RevisionGridControl.Command.GoToParent, Keys.Control | Keys.P),
+                    Hk(RevisionGridControl.Command.NextQuickSearch, Keys.Alt | Keys.Down),
+                    Hk(RevisionGridControl.Command.OpenCommitsWithDifftool, Keys.None),
+                    Hk(RevisionGridControl.Command.PrevQuickSearch, Keys.Alt | Keys.Up),
+                    Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.F),
+                    Hk(RevisionGridControl.Command.SelectCurrentRevision, Keys.Control | Keys.Shift | Keys.C),
+                    Hk(RevisionGridControl.Command.SelectAsBaseToCompare, Keys.Control | Keys.L),
+                    Hk(RevisionGridControl.Command.ShowAllBranches, Keys.Control | Keys.Shift | Keys.A),
+                    Hk(RevisionGridControl.Command.ShowCurrentBranchOnly, Keys.Control | Keys.Shift | Keys.U),
+                    Hk(RevisionGridControl.Command.ShowFilteredBranches, Keys.Control | Keys.Shift | Keys.T),
+                    Hk(RevisionGridControl.Command.ShowFirstParent, Keys.Control | Keys.Shift | Keys.S),
+                    Hk(RevisionGridControl.Command.ShowReflogReferences, Keys.Control | Keys.Shift | Keys.L),
+                    Hk(RevisionGridControl.Command.ShowRemoteBranches, Keys.Control | Keys.Shift | Keys.R),
+                    Hk(RevisionGridControl.Command.ToggleBetweenArtificialAndHeadCommits, Keys.Control | Keys.OemBackslash),
```

Comment:
Alphabetic for Toggle*?
(Was changed for GoTo*)

Reply:
It is sorted. What have I missed?

Reply:
ToggleBetweenArtificialAndHeadCommits
ToggleRevisionGraph
ToggleAuthorDateCommitDate
etc

Since cleanup is done for the other, I suggest here too

Reply:
Yep, raised my eyes above the short-range radar:
![grafik](https://user-images.githubusercontent.com/36601201/79274732-4cb73480-7ea5-11ea-8781-82c5c97f45d7.png)
done